lib/nxt_support/models/email.rb

```diff
@@ -5,8 +5,8 @@ class Email
     #  (2) No whitespace characters
     #  (3) At least one non-whitespace character before the '@' character
     #  (4) No dot ('.') character directly after the '@' character
-    #  (5) A hostname after the '@' character of at least two non-whitespace characters length
+    #  (5) A hostname after the '@' character of at least one non-whitespace characters length
     #  (6) At least one top level domain ending (e.g. '.com') after the hostname, separated from the hostname by a dot ('.')
-    REGEXP = /\A[^@\s]+@[^\.][^@\s]+\.[^@\s]+\z/.freeze
+    REGEXP = /\A[^@\s]+@([^\.^@\s]+\.)+[^@\s]+\z/.freeze
```

Comment:
If we don't want to disallow `^` in domain part, we don't need to write it again before @. Just one `^` in the beginning of the group is enough. (i guess ....) 

Current visualization: https://regexper.com/#%5B%5E%40%5Cs%5D%2B%40%28%5B%5E%5C.%5E%40%5Cs%5D%2B%5C.%29%2B%5B%5E%40%5Cs%5D%2B

```suggestion
    REGEXP = /\A[^@\s]+@([^\.@\s]+\.)+[^@\s]+\z/.freeze
```

Reply:
Good catch! Thanks ❤️